drivers/adc/ad400x/ad400x.c

```diff
@@ -182,6 +182,8 @@ int32_t ad400x_init(struct ad400x_dev **device,
 	if (!dev)
 		return -1;
 
+	dev->ref_voltage_mv = init_param->ref_voltage_mv;
```

Comment:
As I have already mentioned... this parameter doe not have anything to do with the driver, it should be a parameter in the iio_driver only. and you should add it in the same commit in which you add the iio_driver

Reply:
ok